docs/tutorials/dashboard.mdx

```diff
@@ -43,7 +43,7 @@ We started the dashboard and the Jellyfish Media Server separately, so we need t
   but it is meant to authenticate the dashboard to the Jellyfish, so the good practice is to use a token that is hard to guess.
 - **WS/WSS** - whether to use a secure or insecure connection. , it's `ws`, but if the Jellyfish uses secure WebSocket, use `wss`.
 - **HTTP/HTTPS** - whether to use a secure or insecure connection. By default, it's `http`, but if the Jellyfish uses secure HTTP, use `https`.
-- **Server URL** - URL of the Jellyfish server. Here, `localhost:5002` would be a local build, but it can be any URL that points to the Jellyfish.
+- **Server URL** - URL of the Jellyfish server. Here, `localhost:4000` would be a local build, but it can be any URL that points to the Jellyfish.
```

Comment:
Why this change?

Reply:
That's how Jellyfish is run in the tutorial. We export port 4000